test/timeout.js

```diff
@@ -38,27 +56,29 @@ test('timeout option as object', async t => {
 
 test('socket timeout', async t => {
 	const err = await t.throws(got(`${s.url}`, {
-		timeout: {socket: 1},
+		timeout: {socket: 4},
 		retries: 0
 	}));
 
 	t.is(err.code, 'ESOCKETTIMEDOUT');
 });
 
-test('connection, request timeout', async t => {
+test.todo('connection timeout');
+
+test('request timeout', async t => {
 	const err = await t.throws(got(`${s.url}`, {
-		timeout: {socket: 50, request: 1},
+		timeout: {request: 40},
```

Comment:
Can you make `40` a constant at the top of the file and use that instead? Makes it easier if we ever need to change it.